src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -135,33 +178,35 @@ export const DownloadConfirmation: React.FunctionComponent<DownloadConfirmationP
     const [status, setStatus] = useState<StatusEnum>(
       accessToken ? StatusEnum.LOADING_INFO : StatusEnum.SIGNED_OUT,
     )
-    const [state, setState] = useState<DownloadConfirmationState>({
-      fileCount: 0,
-      fileSize: 0,
-    })
-    const { refetch } = useGetDownloadListStatistics()
+
+    const [fileCount, setFileCount] = useState(0)
+    const [fileSize, setFileSize] = useState(0)
+
+    const { data: downloadListStatistics, refetch } =
+      useGetDownloadListStatistics()
 
     const lastQueryRequest = getLastQueryRequest!()
     // is not defined (configured for a container)
     const [showDownloadList, setShowDownloadList] = useState(false)
     const updateStats = useCallback(
-      async (count?: number, bytes?: number) => {
-        if (accessToken) {
-          const estimatedDownloadBytesPerSecond = await testDownloadSpeed(
```

Comment:
The estimated download speed is calculated in the DownloadDetails component. This was stored in state but wasn't otherwise used.